docs/source/user-guide/vision/vision_properties.rst

```diff
@@ -0,0 +1,190 @@
+===============
```

Comment:
This file needs a table of contents. Will one be automatically generated in the docs?